pennylane_forest/device.py

```diff
@@ -361,7 +361,5 @@ def analytic_probability(self, wires=None):
         if self._state is None:
             return None
 
-        wires = wires or range(self.num_wires)
-        wires = Wires(wires)
```

Comment:
``marginal_prob`` is written to return ``prob`` when ``wires is None``, so this (wrong) conversion to ``range(self.num_wires)`` is actually not needed at all - we can just feed the wires argument through this function.